### PR TITLE
feat(scheduler-service): add migration for enums

### PR DESCRIPTION
**Pull request by rashisf**
SFO-98

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [x] Performed a self-review of my own code
- [x] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: services/scheduler-service/migrations/sqls/20200630114107-add-enums-down.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+ALTER TABLE scheduler.subscriptions ALTER COLUMN access_role TYPE varchar;
+
+ALTER TABLE scheduler.attendees ALTER COLUMN response_status TYPE varchar;
+
+ALTER TABLE scheduler.settings ALTER COLUMN owner_type TYPE varchar;
+
+ALTER TABLE scheduler.events ALTER COLUMN status TYPE varchar;
+
+CREATE OR REPLACE VIEW scheduler.events_attendees_view AS
```
Comment (rohit-sourcefuse) [inline services/scheduler-service/migrations/sqls/20200630114107-add-enums-down.sql:13]: @rashisf you have tested this ?

File: services/scheduler-service/migrations/sqls/20200630114107-add-enums-up.sql [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ALTER TABLE scheduler.settings ALTER COLUMN owner_type DROP DEFAULT;
+ALTER TABLE scheduler.settings ALTER COLUMN owner_type TYPE scheduler.setting_owner_type USING owner_type::text::scheduler.setting_owner_type;
+ALTER TABLE scheduler.settings ALTER COLUMN owner_type SET DEFAULT 'global';
+
+CREATE TYPE scheduler.eventStatus AS ENUM ('confirmed', 'tentative', 'cancelled', 'completed');
+ALTER TABLE scheduler.events ALTER COLUMN status DROP DEFAULT;
+ALTER TABLE scheduler.events ALTER COLUMN status TYPE scheduler.eventStatus USING status::text::scheduler.eventStatus;
+ALTER TABLE scheduler.events ALTER COLUMN status SET DEFAULT 'tentative';
+
+CREATE VIEW scheduler.events_attendees_view AS  SELECT et.id,
```
Comment (rohit-sourcefuse) [inline services/scheduler-service/migrations/sqls/20200630114107-add-enums-up.sql:23]: @rashisf you can replace et.* as you have done this in down script

File: services/scheduler-service/migrations/sqls/20200630114107-add-enums-down.sql (hunk 1 above)
Comment (rashisf) [reply services/scheduler-service/migrations/sqls/20200630114107-add-enums-down.sql:13]: yes

File: services/scheduler-service/migrations/sqls/20200630114107-add-enums-up.sql (hunk 2 above)
Comment (rashisf) [reply services/scheduler-service/migrations/sqls/20200630114107-add-enums-up.sql:23]: Okay

**Pull request merged**